src/ucp/core/ucp_ep_vfs.c

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (C) 2021 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "ucp_ep.inl"
+#include "ucp_ep_vfs.h"
+#include "ucp_vfs.h"
+
+#include <ucp/wireup/wireup_ep.h>
+#include <ucs/datastruct/string_buffer.h>
+#include <ucs/sys/sock.h>
+#include <ucs/vfs/base/vfs_cb.h>
+#include <ucs/vfs/base/vfs_obj.h>
+
+
+typedef struct {
+    const char *name;
+    uint64_t   field_mask;
+    size_t     offset;
+} ucp_ep_vfs_attr_t;
+
+
+static const ucp_ep_vfs_attr_t ucp_ep_vfs_attrs[] =
+        {{"local", UCP_EP_ATTR_FIELD_LOCAL_SOCKADDR,
+          ucs_offsetof(ucp_ep_attr_t, local_sockaddr)},
+         {"remote", UCP_EP_ATTR_FIELD_REMOTE_SOCKADDR,
+          ucs_offsetof(ucp_ep_attr_t, remote_sockaddr)}};
```

Comment:
weird align

Reply:
It was suggestion of code style checker.

Reply:
let's ignore it in this case